.eslintrc.yml

```diff
@@ -0,0 +1,12 @@
+env:
+  es6: true
+  node: true
+  mocha: true
+extends:
+  - eslint:recommended
+  - airbnb-base
+  - plugin:import/errors
+  - plugin:import/warnings
+rules:
+  "comma-dangle": ["error", "never"]
+  
```

Comment:
To fix the EOF, you should make sure the line 12 is totally empty, without tab or whitespace, the warning is still there:
![image](https://user-images.githubusercontent.com/10209180/177195072-40909e98-64bd-49e1-80f4-c2fff00e972e.png)

Remember that you can configure your visual studio code to add automatically the EOF: https://stackoverflow.com/a/44704969